tests/error/misc_errors/return_flag_callable.py

```diff
@@ -11,7 +11,7 @@
 
 
 @guppy.declare(module)
-def foo(f: Callable[[], qubit @inout]) -> None: ...
+def foo(f: "Callable[[], qubit @inout]") -> None: ...
```

Comment:
This is a bit annyoing. The `qubit` type now needs to be a `Definition` so we can import it, however Pythons `Callable` checks that the generic args are `type`s at runtime...

Fixing this with a string annotation for now, but should figure out something better. See #433